common/artifact-manager/src/main/java/com/vmware/pscoe/iac/artifact/store/vrang/VraNgPropertyGroupStore.java

```diff
@@ -131,142 +182,137 @@ protected List<String> getItemListFromDescriptor() {
-				item -> item.getName().equalsIgnoreCase( propertyGroupName )
-			).findFirst().orElse(null);
+				item -> item.getName().equalsIgnoreCase(propertyGroupName)).findFirst().orElse(null);
 	}
 
 	/**
-	 * Stores teh property group on the file system
+	 * Stores the property group on the file system.
 	 *
-	 * @param	serverPackage
-	 * @param	propertyGroup
+	 * @param serverPackage
+	 * @param propertyGroup
 	 */
-	private void storePropertyGroupOnFileSystem( Package serverPackage, VraNgPropertyGroup propertyGroup ) {
-		logger.info( "Storing PropertyGroup: {}", propertyGroup.getName() );
+	private void storePropertyGroupOnFileSystem(Package serverPackage, VraNgPropertyGroup propertyGroup) {
+		logger.info("Storing PropertyGroup: {}", propertyGroup.getName());
 
-		File store						= new File( serverPackage.getFilesystemPath() );
-		File customPropertyGroupFile	= Paths.get(
-			store.getPath(),
-			VraNgDirs.DIR_PROPERTY_GROUPS,
-			propertyGroup.getName() + PROPERTY_GROUP_SUFFIX
-		).toFile();
+		File store = new File(serverPackage.getFilesystemPath());
+		File customPropertyGroupFile = Paths.get(
+				store.getPath(),
+				VraNgDirs.DIR_PROPERTY_GROUPS,
+				propertyGroup.getName() + PROPERTY_GROUP_SUFFIX).toFile();
 
-		logger.debug( "Creating folder: {}", customPropertyGroupFile.getParentFile().getAbsolutePath() );
+		logger.debug("Creating folder: {}", customPropertyGroupFile.getParentFile().getAbsolutePath());
 
-		if ( ! customPropertyGroupFile.getParentFile().mkdirs() ) {
-			logger.warn( "Could not create folder: {}", customPropertyGroupFile.getParentFile().getAbsolutePath() );
+		if (!customPropertyGroupFile.getParentFile().mkdirs()) {
+			logger.warn("Could not create folder: {}", customPropertyGroupFile.getParentFile().getAbsolutePath());
 		}
 
 		try {
-			Gson gson	= new GsonBuilder().setLenient().setPrettyPrinting().serializeNulls().create();
+			Gson gson = new GsonBuilder().setLenient().setPrettyPrinting().serializeNulls().create();
 			final JsonObject propertyGroupJsonElement = gson.fromJson(propertyGroup.getRawData(), JsonObject.class);
 
-            this.sanitizePropertyGroupJsonElement(propertyGroupJsonElement);
+			this.sanitizePropertyGroupJsonElement(propertyGroupJsonElement);
 
-            String propertyGroupJson = gson.toJson(propertyGroupJsonElement);
-            logger.info("Created file {}", Files.write(
-				Paths.get(customPropertyGroupFile.getPath()),
-				propertyGroupJson.getBytes(), 
-				StandardOpenOption.CREATE
-			));
-		} catch ( IOException e ) {
-			logger.error( "Unable to create property group {}", customPropertyGroupFile.getPath() );
+			String propertyGroupJson = gson.toJson(propertyGroupJsonElement);
+			logger.info("Created file {}", Files.write(
+					Paths.get(customPropertyGroupFile.getPath()),
+					propertyGroupJson.getBytes(),
+					StandardOpenOption.CREATE));
+		} catch (IOException e) {
+			logger.error("Unable to create property group {}", customPropertyGroupFile.getPath());
```

Comment:
Logging seems redundant here, won't the thrown error be logged?

Reply:
These are only automatic formatter changes. Logging was there before.